README.md

```diff
@@ -26,35 +26,69 @@ npm install --save-dev eslint eslint-plugin-n
 
 **Note:** It recommends a use of [the "engines" field of package.json](https://docs.npmjs.com/files/package.json#engines). The "engines" field is used by `n/no-unsupported-features/*` rules.
 
-### **[.eslintrc.json](https://eslint.org/docs/latest/use/configure/configuration-files)** (An example)
+### [`eslint.config.js`](https://eslint.org/docs/latest/use/configure/configuration-files-new) (requires eslint>=v8.23.0)
 
-```jsonc
-{
-    "extends": ["eslint:recommended", "plugin:n/recommended"],
-    "parserOptions": {
-        "ecmaVersion": 2021
-    },
-    "rules": {
-        "n/exports-style": ["error", "module.exports"]
+```js
+const nodePlugin = require("eslint-plugin-n")
+
+module.exports = [
+    nodePlugin.configs["flat/recommended-script"],
+    {
+        rules: {
+            "n/exports-style": ["error", "module.exports"]
+        }
     }
-}
+]
 ```
 
-### [`eslint.config.js`](https://eslint.org/docs/latest/use/configure/configuration-files-new) (requires eslint>=v8.23.0)
+To setup without the recommended configs, you'll need to add the plugin:
 
 ```js
 const nodePlugin = require("eslint-plugin-n")
 
 module.exports = [
-    nodePlugin.configs["flat/recommended-script"],
     {
+		plugins: {
+			n: node
+		},
         rules: {
             "n/exports-style": ["error", "module.exports"]
         }
     }
 ]
 ```
 
+### **[.eslintrc.json](https://eslint.org/docs/latest/use/configure/configuration-files)** (legacy example)
+
+```jsonc
+{
+    "extends": ["eslint:recommended", "plugin:n/recommended"],
+    "parserOptions": {
+        "ecmaVersion": 2021
+    },
+    "rules": {
+        "n/exports-style": ["error", "module.exports"]
+    }
+}
+```
+
+To setup without the recommended rules you'll need to add the plugin:
+
+```jsonc
+{
+    "parserOptions": {
+        "ecmaVersion": 2021
+    },
+    "plugins": {
+		"n": node
+    },
```

Comment:
Im not sure this'll work in the legacy version?

Reply:
I don't know. I only have the new version to test on. We can remove this section